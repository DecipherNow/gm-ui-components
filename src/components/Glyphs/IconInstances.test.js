import React from "react";
import { shallow } from "enzyme";

import IconInstances from "./IconInstances";

describe("IconInstances", () => {
  it("matches snapshot", () => {
    const aIconInstances = shallow(<IconInstances />);
    expect(aIconInstances).toMatchSnapshot();
  });
});