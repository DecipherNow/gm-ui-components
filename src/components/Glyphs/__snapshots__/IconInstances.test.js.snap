// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IconInstances matches snapshot 1`] = `
<Icon
  backgroundColor="currentColor"
  backgroundOpacity={1}
  backgroundSizeRatio={1}
  borderColor="currentColor"
  borderOpacity={1}
  borderWidth={1}
  glyph=""
  glyphColor="currentColor"
  glyphName="Instances"
  glyphSizeRatio={1}
>
  <path
    d="M5.99999997,7.99999998 C4.89543047,7.99999998 3.99999997,7.10456948 3.99999997,5.99999997 C3.99999997,4.89543047 4.89543047,3.99999997 5.99999997,3.99999997 C7.10456948,3.99999997 7.99999998,4.89543047 7.99999998,5.99999997 C7.99999998,7.10456948 7.10456948,7.99999998 5.99999997,7.99999998 Z M9.99999999,12 C8.89543049,12 7.99999998,11.1045695 7.99999998,9.99999999 C7.99999998,8.89543049 8.89543049,7.99999998 9.99999999,7.99999998 C11.1045695,7.99999998 12,8.89543049 12,9.99999999 C12,11.1045695 11.1045695,12 9.99999999,12 Z M14,16 C12.8954305,16 12,15.1045695 12,14 C12,12.8954305 12.8954305,12 14,12 C15.1045695,12 16,12.8954305 16,14 C16,15.1045695 15.1045695,16 14,16 Z M18,20 C16.8954305,20 16,19.1045695 16,18 C16,16.8954305 16.8954305,16 18,16 C19.1045695,16 20,16.8954305 20,18 C20,19.1045695 19.1045695,20 18,20 Z"
    fillRule="nonzero"
  />
</Icon>
`;
